### PR TITLE
Dafne

### PR DESCRIPTION
**Pull request by dafnevk**
Added pages for Dafne, Elena, and Salient region detectors.

### CODE REVIEW

File: _software/salient-region-detectors.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+competence:
+- Big Data Analytics
+discipline:
+- eScience Methodology
+expertise:
+- Computer Vision
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/e.ranguelova
+owner:
+- http://software.esciencecenter.nl/organization/nlesc/
```
Comment (blootsvoets) [inline _software/salient-region-detectors.md]: remove slash at the end

File: _software/salient-region-detectors.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+license:
+- apache-2.0
+competence:
+- Big Data Analytics
+discipline:
+- eScience Methodology
+expertise:
+- Computer Vision
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/e.ranguelova
```
Comment (blootsvoets) [inline _software/salient-region-detectors.md]: add contact person of

**Comment by blootsvoets** [comment]
@elboyran could you review

**Comment by elboyran** [comment]
I'm not sure how to do this, I wasn't at the hands-on session. Maybe tomorrow I can talk to whoever is in the center (best- a reviewer ;-).

**Comment by elboyran** [comment]
In the project description I would use singular "Software package", not "packages" as it is now. Maybe also it's better to use "detector/detection" rather than "finding". 

I can't seem to open the link of the project, i..e I get 404 File Not Found trying http://software.esciencecenter.nl/software/salient-region-detectors
So, I actually can't review the project page at the moment.

For the people's pages: for my page I would edit the last sentence: "Elena joined..." after "to work on"
and replace the remainder of the sentence with "multidisciplinary projects  with computer vision and machine learning components".
Also from my page there should be a link to Biomarker boosting project!

Dafne's page: Would be nice to have a photo :-). A link not only to the Salient regions, but also to the protein's project?

**Comment by dafnevk** [comment]
* WIP means Work in Progress (see http://www.repostatus.org). We haven't released the package yet, that's why I put WIP.
* I changed the words to 'package' , although I would say this is actually specific for python. The software is also implemented in Matlab.
* Thre is no page for the biomarker boosting project, I let it up to someone else to add that.
* I will put my photo up as soon as I get a photo for the website.
* There is no project yet for the Protein Bindings, and I don't think we will put it there in this early stage. 

The other comments I took care of. The branch now passes the validation.

**Pull request merged**